Build failing due to wrong SPIRV-Headers DEPS revision

ANGLE is hitting a build error trying to build SPIRV-Tools related to the wrong revision of SPIRV-Headers being specified in the DEPS file.

```
../../third_party/spirv-tools/src/source/opt/code_sink.cpp:217:12: error: use of undeclared identifier 'SpvOpAtomicFAddEXT'; did you mean 'SpvOpAtomicIAdd'?
      case SpvOpAtomicFAddEXT:
```

https://github.com/KhronosGroup/SPIRV-Tools/blob/f3cec93665ec6097dbc56dafbb8b46595842eded/DEPS#L9

The symbol `SpvOpAtomicFAddEXT` was added after this revision (I haven't looked which revision, but 979924c8bc839e4cb1b69d03d48398551f369ce7 looks good).

--- a/DEPS
+++ b/DEPS
@@ -3,10 +3,10 @@ use_relative_paths = True
 vars = {
   'github': 'https://github.com',
 
-  'effcee_revision': '5af957bbfc7da4e9f7aa8cac11379fa36dd79b84',
-  'googletest_revision': '011959aafddcd30611003de96cfd8d7a7685c700',
-  're2_revision': 'aecba11114cf1fac5497aeb844b6966106de3eb6',
-  'spirv_headers_revision': 'ac638f1815425403e946d0ab78bac71d2bdbf3be',
+  'effcee_revision': '2ec8f8738118cc483b67c04a759fee53496c5659',
+  'googletest_revision': 'a781fe29bcf73003559a3583167fe3d647518464',
+  're2_revision': 'ca11026a032ce2a3de4b3c389ee53d2bdc8794d6',
+  'spirv_headers_revision': '979924c8bc839e4cb1b69d03d48398551f369ce7',
 }
 
 deps = {

